### ISSUE TITLE
Accesibility : option to add role tooltip and id

### ISSUE THREAD

**Issue by monir-a-khan** [drozhzhin-n-e/ng2-tooltip-directive#167]
According to w3c, the tooltip should have role="tooltip". Also an id, the triggering element should have aria-describedby which will refer to this id. 

Example scenario which should be achieved is

```
<button class="notifications" aria-labelledby="notifications-label"> Show tooltip
</button>  
<div role="tooltip" id="notifications-label">A tooltip text</div>  
```
